miasm2/ir/__init__.py

```diff
@@ -0,0 +1 @@
+"Intermediate reprensentation methods"
```

Comment:
Typo: reprensentation -> representation
Maybe add that it also contains the symbolic execution methods?